### DISCUSSION TITLE
При создании новой записи дата выбирается только до 2020 года.

### DISCUSSION THREAD

**Discussion by aktsh0** [maxsite/cms#464] [General]
Версия MaxSite CMS: 0.863 При создании новой записи невозможно указать год. В выпадающем меню доступно для выбора до 2020 г. Из этого запись уходит в конец списка и её не видно в новостях. 
**Помогите решить проблему.** Обнаружилось только что. В принципе ничего не менялось и не добавлялось уже много лет. 
Сайт работает исправно более 6 лет.

**Comment by maxsite** [comment]
Обновите MaxSite CMS.
https://github.com/maxsite/cms/issues/424
